src/bereikbaarheid/resources/utils.py

```diff
@@ -107,15 +107,22 @@ def refresh_materialized(db_table: str):
 # -------------------------------------------
 
 
-def convert_to_date(date: str = None, format: str = "%d/%m/%y %H:%M"):
+def convert_to_date(date: str = None, format: str = "%d/%m/%y %H:%M") -> datetime:
     """Convert string format to datetime"""
 
+
     try:
         _date = datetime.datetime.strptime(date, format)
+        return _date
     except ValueError:
-        print(f"verkeerd datumformat voor {date}, gewenst format is {format}")
+        try:
+            format = "%Y-%m-%d %H:%M:%S.%f"
+            _date = datetime.datetime.strptime(date, format)
+            return _date
+        except ValueError:
+            print(f"verkeerd datumformat voor {date}, gewenst format is {format}")
```

Comment:
Als we een runnend programma hebben dan gebruik je eigenlijk nooit print, tenzij het echt een cli is. In dit geval zou je een log gebruiken zoals bv [hier](https://github.com/Amsterdam/bereikbaarheid-backend-django/blob/main/src/health/views.py#L16).

Reply:
Overigens kan je die string ook gewoon in de ValueError kwijt die je raised. Dus dan hoeft er helemaal geen log te zijn. Een Error wordt immers ook al opgepakt door de logging.